src/components/_pages/certificates/detail/index.tsx

```diff
@@ -1110,10 +1176,10 @@ export default function CertificateDetail() {
                       columns: ["Basic Constraint", certificate.basicConstraints],
                   },
               ];
-        if (health && certificate?.status !== CertStatus.New) {
+        if (certificate?.status !== CertStatus.New) {
```

Comment:
why "health" condition is removed?

Reply:
keeping it to show as discussed during our team call..